simplivity/resources/policies.py

```diff
@@ -199,3 +199,42 @@ def get_rule(self, rule_id):
         """
         resource_uri = "{}/{}/rules/{}".format(URL, self.data["id"], rule_id)
         return self._client.do_get(resource_uri)
+
+    def edit_rule(self, rule_id, rule, timeout=-1):
+        """Edits an existing policy rule
+        Args:
+            rule_id: Rule id to be edited
+            rule: Any or all of the backup policy rule fields can change, except destination_name, id, number,
```

Comment:
mention the expected type of this argument

Reply:
Updated as suggested